cmd/broker/ingress/metrics.go

```diff
@@ -0,0 +1,86 @@
+/*
+ * Copyright 2019 The Knative Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package main
+
+import (
+	"go.opencensus.io/stats"
+	"go.opencensus.io/stats/view"
+	"go.opencensus.io/tag"
+)
+
+const (
+	metricsNamespace = "broker_ingress"
```

Comment:
Is it worth including knative?

Reply:
Not sure if it's correct, but the prometheus exporter in knative/pkg as used by knative/serving doesn't. The prometheus namespace is just the component name, for example `controller`.

https://github.com/knative/serving/blob/c5d12355921c32c4c140e3e3cc12db6a31868523/cmd/controller/main.go#L59
https://github.com/knative/serving/blob/c5d12355921c32c4c140e3e3cc12db6a31868523/cmd/controller/main.go#L203
https://github.com/knative/pkg/blob/1ed7b993c877a0d1914574344e500f5c99909086/metrics/prometheus_exporter.go#L31